octobot_trading/personal_data/exchange_personal_data.py

```diff
@@ -218,6 +218,9 @@ async def handle_order_update_notification(self, order, is_new_order):
         :param is_new_order: True if the order was created during update
         """
         try:
+            chan = exchange_channel.get_chan(constants.ORDERS_CHANNEL, self.exchange_manager.id)
+            if not chan.get_consumers():
+                return
```

Comment:
Should we implement it in `send` and add a param in `send()` like `only_if_having_consumer`?

Reply:
here the idea is to avoid doing the "to_dict" call when not necessary. I don't think we should change how send() works

Reply:
Oh yes I see. That's better like this